packages/app/components/Queue/QueueListSharedComponents.tsx

```diff
@@ -1,126 +1,122 @@
-import { ClockCircleFilled, SettingOutlined } from "@ant-design/icons";
-import { API } from "@template/api-client";
-import { Avatar, Input, Switch, Tooltip } from "antd";
-import Modal from "antd/lib/modal/Modal";
-import { ReactElement, useState } from "react";
+import { ClockCircleOutlined, NotificationOutlined } from "@ant-design/icons";
+import React, { ReactElement, ReactNode } from "react";
 import styled from "styled-components";
 import { useQueue } from "../../hooks/useQueue";
 import { formatQueueTime } from "../../utils/TimeUtil";
+import { TAStatuses } from "./TAStatuses";
+import { Button } from "antd";
+import { ButtonProps } from "antd/lib/button";
+
+export const QueuePageContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  @media (min-width: 767px) {
+    flex-direction: row;
+  }
+`;
+
+export const VerticalDivider = styled.div`
```

Comment:
If possible, could we get this divider to extend for the full height of the page instead of cutting off at where the content ends?